### ISSUE TITLE
Composer dependency issue - reflection-docblock

### ISSUE THREAD

**Issue by ghostal** [ApiGen/ApiGen#1000]
This is a similar-looking issue to #988 but with a different package.

Here is my `composer.json` (I'm using this as part of [nanoninja's docker lemp setup](https://github.com/nanoninja/docker-nginx-php-mysql)):

    {
        "require": {
    
        },
        "require-dev": {
            "apigen/apigen": "^5.0.0-RC4"
        },
        "autoload": {
            "psr-4": {
                "App\\Acme\\": "src/"
            }
        },
        "autoload-dev": {
            "psr-4": {
                "AppTest\\Acme\\": "test/"
            }
        },
        "minimum-stability": "dev",
        "prefer-stable": true
    }

And here's the output from composer:

    Loading composer repositories with package information
    Updating dependencies (including require-dev)
    Your requirements could not be resolved to an installable set of packages.

      Problem 1
        - Conclusion: don't install apigen/apigen v5.0.0-RC5
        - apigen/apigen v5.0.0-RC4 requires phpdocumentor/reflection-docblock ^3.1 -> satisfiable by phpdocumentor/reflection-docblock[3.1.0, 3.1.1, 3.2.0, 3.2.1, 3.2.2, 3.2.3].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.1.0].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.1.1].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.0].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.1].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.2].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.3].
        - roave/better-reflection dev-master requires phpdocumentor/reflection-docblock ^4.1.1 -> satisfiable by phpdocumentor/reflection-docblock[4.1.1].
        - apigen/apigen v5.0.0-RC4 requires roave/better-reflection dev-master#a83ef24 -> satisfiable by roave/better-reflection[dev-master].
        - Installation request for apigen/apigen ^5.0.0-RC4 -> satisfiable by apigen/apigen[v5.0.0-RC4, v5.0.0-RC5].

I noticed that RC5 is available now, so tried updating my `composer.json` to specify the new RC, but got the same issue:

    Loading composer repositories with package information
    Updating dependencies (including require-dev)
    Your requirements could not be resolved to an installable set of packages.

      Problem 1
        - apigen/apigen v5.0.0-RC5 requires phpdocumentor/reflection-docblock ^3.1 -> satisfiable by phpdocumentor/reflection-docblock[3.1.0, 3.1.1, 3.2.0, 3.2.1, 3.2.2, 3.2.3].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.1.0].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.1.1].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.0].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.1].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.2].
        - Can only install one of: phpdocumentor/reflection-docblock[4.1.1, 3.2.3].
        - roave/better-reflection dev-master requires phpdocumentor/reflection-docblock ^4.1.1 -> satisfiable by phpdocumentor/reflection-docblock[4.1.1].
        - apigen/apigen v5.0.0-RC5 requires roave/better-reflection dev-master#a402048 -> satisfiable by roave/better-reflection[dev-master].
        - Installation request for apigen/apigen ^5.0.0-RC5 -> satisfiable by apigen/apigen[v5.0.0-RC5].

**Comment by vlastavesely** [comment]
Hi, could you test it on `dev-master` ApiGen? I have made changes so it should be fixed.

**Comment by ghostal** [comment]
Hi @vlastavesely, I can confirm using `dev-master` results in a successful install. Still getting an error as follows, but guessing it's an unreleated problem, so feel free to close this issue if you agree :smiley: 

    # php vendor/bin/apigen
    
    Fatal error: Uncaught Error: Class 'Roave\BetterReflection\SourceLocator\Ast\PhpParserLocator' not found in /tmp/_apigen_kernel_cache/DependencyInjectionDevDebugProjectContainer.php on line 1451
    
    Error: Class 'Roave\BetterReflection\SourceLocator\Ast\PhpParserLocator' not found in /tmp/_apigen_kernel_cache/DependencyInjectionDevDebugProjectContainer.php on line 1451

**Comment by vlastavesely** [comment]
There are changes being made with dependency on `BetterReflection`. It may be solved after #992 . If not, open new issue please.

**Comment by nanoninja** [comment]
Using the following versions it works for me
  
```json
"require-dev": {
    "squizlabs/php_codesniffer": "3.*",
    "phpunit/phpunit": "^6.0",
    "apigen/apigen": "dev-master",
    "roave/better-reflection": "dev-master#a402048"
 }
```
